schedule/yast/autoyast/autoyast_btrfs_quota.yaml

```diff
@@ -0,0 +1,89 @@
+---
+name: autoyast_btrfs_quota
+description: >
+  AutoYaST installation with BTRFS subvolume quotas. Verifies quota limit on the installed system
+  and if its size corresponds to the configured quota limit during installation.
+vars:
+  AUTOYAST_PREPARE_PROFILE: 1
+  DESKTOP: textmode
```

Comment:
Will we end up in textmode for openSUSE? I guess this setting will differ for SLES and TW, for example.

Reply:
Yes, the installed system is in textmode. Is it an issue? Do we have any restrictions or agreements against that?

Reply:
Nope, from the profile it's not visible what will be there for openSUSE, but from VR I can see that no WM is installed: https://openqa.opensuse.org/tests/1630033#step/installation/8 so we are good here. Thanks!

Reply:
Ok. Then the VR with the latest updates for TW has passed. I guess the PR could be merged?:) https://openqa.opensuse.org/tests/1630050

Reply:
Yes, please, take care of the relevant job group settings on osd and o3.